TRModelTransporter/Packing/TexturedTile.cs

```diff
@@ -1,163 +1,155 @@
         }
+    }
 
-        protected override bool Add(TexturedTileSegment segment, int x, int y)
+    protected override bool Add(TexturedTileSegment segment, int x, int y)
+    {
+        bool added = base.Add(segment, x, y);
+        if (added)
         {
-            bool added = base.Add(segment, x, y);
-            if (added)
-            {
-                CheckBitmapStatus();
-                BitmapGraphics.Import(segment.Bitmap, segment.MappedBounds);
+            CheckBitmapStatus();
+            BitmapGraphics.Import(segment.Bitmap, segment.MappedBounds);
 
-                segment.Bind();
-                _segmentAdded = true;
-            }
-            return added;
+            segment.Bind();
+            _segmentAdded = true;
         }
+        return added;
+    }
 
-        public override bool Remove(TexturedTileSegment segment)
+    public override bool Remove(TexturedTileSegment segment)
+    {
+        bool removed = base.Remove(segment);
+        if (removed)
         {
-            bool removed = base.Remove(segment);
-            if (removed)
-            {
-                CheckBitmapStatus();
-                BitmapGraphics.Delete(segment.Bounds);
-                segment.Unbind();
-                _segmentRemoved = true;
-            }
-            return removed;
+            CheckBitmapStatus();
+            BitmapGraphics.Delete(segment.Bounds);
+            segment.Unbind();
+            _segmentRemoved = true;
         }
+        return removed;
+    }
 
-        public override void PackingStarted()
-        {
-            // No overlapping during actual packing
-            AllowOverlapping = false;
-
-            // Only if a segment has been removed prior to packing having
-            // started do we keep a hold of that fact. This prevents the
-            // bitmap being saved back to the level if it's unchanged.
-            if (!_segmentRemoved)
-            {
-                _segmentAdded = _segmentRemoved = false;
-            }
-        }
+    public override void PackingStarted()
+    {
+        // No overlapping during actual packing
+        AllowOverlapping = false;
 
-        private void CheckBitmapStatus()
+        // Only if a segment has been removed prior to packing having
+        // started do we keep a hold of that fact. This prevents the
+        // bitmap being saved back to the level if it's unchanged.
+        if (!_segmentRemoved)
         {
-            if (BitmapGraphics == null)
-            {
-                BitmapGraphics = new BitmapGraphics(new Bitmap(Width, Height, PixelFormat.Format32bppArgb));
-            }
+            _segmentAdded = _segmentRemoved = false;
         }
+    }
 
-        public void Dispose()
-        {
-            if (BitmapGraphics != null)
-            {
-                BitmapGraphics.Dispose();
-            }
+    private void CheckBitmapStatus()
+    {
+        BitmapGraphics ??= new(new Bitmap(Width, Height, PixelFormat.Format32bppArgb));
```

Comment:
My intuition, which is outside of this PR's scope, is that if the object B lifecycle is tied to A, it should A's responsibility to create B, rather than be given B by C.
Eg instead of:

```c#
public class BitmapManager : IDisposable {
    public Bitmap Bitmap { get; set; }

    public BitmapManager(Bitmap bitmap) {
        Bitmap = bitmap;
    }

    public Dispose() {
        Bitmap.Dispose();
        GC.SupressFinalize();
    }
}

using BitmapManager bitmapManager = new(new Bitmap("foo.bmp"));
```

do

```C#
public class BitmapManager : IDisposable {
    public Bitmap Bitmap { get; set; }

    public BitmapManager(string path) {
        Bitmap = new Bitmap(path);
    }

    public Dispose() {
        Bitmap.Dispose();
        GC.SupressFinalize();
    }
}

using BitmapManager bitmapManager = new("foo.bmp");
```